test/test.bash

```diff
@@ -13,11 +13,11 @@ PATH="${DIR}/tmp/test/bin:$PATH"
 cp test/buf.bash tmp/test/bin/buf
 chmod +x tmp/test/bin/buf
 
-# prevent the GITHUB_SHA set by actions from being used in test
-unset GITHUB_SHA
+# prevent the GITHUB_SHA, GITHUB_REF_NAME and GITHUB_REF_TYPE set by actions from being used in test
```

Comment:
Does this prevent us from writing a test?

Reply:
I am not 100% remember, but I believe this is because the test in CI will have these environment variables set and affect the test result